components/adapters/ContentBlockAdapter.tsx

```diff
@@ -67,9 +76,10 @@ export const ContentBlockAdapter: React.FC<ContentBlockAdapterProps> = ({
         {title && <h2>{title}</h2>}
         {description?.json &&
           documentToReactComponents(
-            description.json,
-            getRichTextRenderOptions(description.links)
+            description.json
+            // getRichTextRenderOptions(description.links)
```

Comment:
is this intended?

Reply:
Yeah i've left it there for now and the function above with the comment.